addons/web/static/tests/chrome/action_manager_tests.js

```diff
@@ -1606,8 +1612,8 @@ QUnit.module('ActionManager', {
             'load_views', // action 3
             '/web/dataset/search_read', // search read of list view of action 3
             '/web/action/load', // action 4
-            'load_views', // action 4
```

Comment:
TODO

Reply:
getController has been simplified a lot (no need to wait for ControlPanel to be ready to load model). The searchQuery is already ready in init (but we wait for getController to call updateMCCParams because some views do strange init overrides).